test/jdk/java/lang/ref/CleanerTest.java

```diff
@@ -567,6 +566,6 @@ public void testCleanerFactory() {
         CleanableCase s = setupPhantom(cleaner, obj);
         obj = null;
         checkCleaned(s.getSemaphore(), true,
-                "Object was cleaned using CleanerFactor.cleaner():");
+                "Object was cleaned using CleanerFactory.cleaner():");
```

Comment:
So the message in case of failure for this case will now be?
```
Should have been run: Object was cleaned using CleanerFactory.cleaner():
```

Reply:
Yes, (just a typo fix); but the message describes the function of the cleanup
and is prefixed in the error as to whether it was run or not.